src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -3427,6 +3427,28 @@ void clang_c_convertert::get_decl_name(
     }
     break;
 
+  case clang::Decl::CXXConstructor:
+    if (name.empty())
+    {
+      // Anonymous constructor, generate a name based on the type
```

Comment:
we don't use its type to generate the name, only its location. Please fix comment